origins_workflow/origins_workflow.module

```diff
@@ -85,3 +86,38 @@ function origins_workflow_form_alter(&$form, FormStateInterface $form_state, $fo
     }
   }
 }
+
+/**
+ * Implements hook_views_pre_render().
+ */
+function origins_workflow_views_pre_render(ViewExecutable $view) {
+  // Hide one of the fields to set moderation state depending
+  // on whether the user has 'publish' permission or not.
+  if ($view->element['#name'] == 'workflow_moderation') {
+    $account_uid = \Drupal::currentUser()->id();
+    $account = User::load($account_uid);
+    if (($view->element['#display_id'] == 'page_2')
```

Comment:
Not a blocking issue, but it would be useful to replace the placeholder display ids with something descriptive. It's easily overlooked but does help when reviewing code again in future.

Reply:
Second to this, it's a right PITA if you later need to rely on this naming structure in code.

Reply:
Fair enough, have done that now